ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -747,17 +751,17 @@ def generate_from_vlan(self):
 
     def generate_ping_dut_lo(self):
         self.ping_dut_packets = []
-        dut_lo_ipv4 = self.test_params['lo_prefix'].split('/')[0]
+        dut_lo_ipv4 = self.lo_prefix.split('/')[0]
         for src_port in self.vlan_host_ping_map:
             src_addr = random.choice(self.vlan_host_ping_map[src_port].keys())
             src_mac = self.hex_to_mac(self.vlan_host_ping_map[src_port][src_addr])
-            packet = simple_icmp_packet(eth_src=src_mac,
-                                        eth_dst=self.dut_mac,
+            packet = simple_icmp_packet(eth_src=self.from_server_src_mac,
```

Comment:
Why is this change needed for src_mac? src_mac in this change becomes same for all the servers, but that's not true right?

Reply:
Correct this change is not needed, fixed!